core/src/main/java/org/apache/accumulo/testing/core/randomwalk/shard/BulkInsert.java

```diff
@@ -172,7 +171,7 @@ private void sort(State state, RandWalkEnv env, FileSystem fs, String tableName,
 
     Collection<Text> splits = conn.tableOperations().listSplits(tableName, maxSplits);
     for (Text split : splits)
-      out.println(Base64.getEncoder().encodeToString(TextUtil.getBytes(split)));
+      out.println(Base64.getEncoder().encodeToString(split.getBytes()));
```

Comment:
Be careful here.  Text has length.  The getBytes() method may return the internal byte array, which may be longer than the length.  I think this is the reason that TextUtil.getBytes() exists.

Reply:
This case and a few others made me wonder why code from util is not part of the API?

Reply:
Well, ideally, Text will not be part of the API, eventually. It might make sense to move a few helpful utils into the public API in the interim.

Reply:
Text has a copyBytes() method.  Would probably be best to use that here.  I think TextUtil method only copies if needed.